kato/kato-azure/src/main/groovy/com/netflix/spinnaker/kato/azure/deploy/converters/AzureAtomicOperationConverterHelper.groovy

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015 The original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.kato.azure.deploy.converters
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.netflix.spinnaker.kato.security.AbstractAtomicOperationsCredentialsSupport
+
+class AzureAtomicOperationConverterHelper {
+  static Object convertDescription(Map input,
+                                   AbstractAtomicOperationsCredentialsSupport credentialsSupport,
+                                   Class targetDescriptionType) {
+    input.accountName = input.credentials
+    // The value returned by getCredentialsObject() is of type AzureNamedAccountCredentials. The credentials property
```

Comment:
it's pretty much a straight copy for the moment; when I copy-and-paste it I was not sure if we have to extend it to be more Azure specific (it gives a change to alter the input before being converted into the final description object). But I think we should be able to call the Google's one; I made a note for myself to remove this once all the other operations are implemented and if there's nothing else that is Azure specific that needs to be added.